src/com/facebook/buck/jvm/groovy/GroovycStep.java

```diff
@@ -91,14 +98,61 @@ public String getDescription(ExecutionContext context) {
   }
 
   private ImmutableList<String> createCommand() {
-    ImmutableList.Builder<String> command = ImmutableList.builder();
-    command.addAll(groovyc.getCommandPrefix(resolver))
+    final ImmutableList.Builder<String> command = ImmutableList.builder();
+
+    command.addAll(groovyc.getCommandPrefix(resolver));
+
+    String classpath = Joiner.on(":").join(transform(declaredClasspathEntries, toStringFunction()));
```

Comment:
Again, this isn't wrong in this diff since you just moved the code, but we shouldn't hardcode `:` here, but instead use `File.pathSeparator` so this works on Windows too.